rts/Sim/Misc/TeamHandler.h

```diff
@@ -107,6 +107,23 @@ class CTeamHandler
 	 */
 	void SetAlly(int allyteamA, int allyteamB, bool allied) { allyTeams[allyteamA].allies[allyteamB] = allied; }
 
+	/**
+	 * @brief set ally team start box
+	 * @param allyteam ally team to edit
+	 * @param startRectLeft left start box boundary
+	 * @param startRectTop top start box boundary
+	 * @param startRectRight right start box boundary
+	 * @param startRectBottom bottom start box boundary
```

Comment:
document range: 0-1 (since it's different from the lua api)

Reply:
Fixed.